runtime/src/pcr.rs

```diff
@@ -22,6 +23,7 @@ use zerocopy::FromBytes;
 
 pub struct IncrementPcrResetCounterCmd;
 impl IncrementPcrResetCounterCmd {
+    #[cfg_attr(not(feature = "no-cfi"), cfi_impl_fn)]
```

Comment:
I think we also probably want CFI on the following mailbox commands:

* FIPS selftest
* FIPS shutdown
* disable attestation

Reply:
Adding CFI to any of these qill require a refactor so I was planning on doing it in the followup PR.

Reply:
I went back and found that the panic test was failing due to not adding #[inline(never)] in a few places. So now CFI should be complete in RT with this PR (i.e. added to disable attestation, fips stuff, among other places).